Sources/Drop.swift

```diff
@@ -58,3 +63,20 @@ public struct Drop {
     public let icon: UIImage?
     public let action: Action?
 }
+
+// MARK: - Equatable Functions
+extension Drop: Equatable {
```

Comment:
Why conforming to `Equatable`?

Reply:
https://github.com/omaralbeik/drops/blob/d756654eb8e551806841540d4fcf8d4acdbbc8f7/Example/ViewController.swift#L84

Because I defined a special time here, I had to provide a presentation by comparing the drop model.